src/app/zap-templates/zcl/data-model/chip/pressure-measurement-cluster.xml

```diff
@@ -32,7 +32,7 @@ limitations under the License.
     <attribute side="server" code="0x0010" define="PRESSURE_SCALED_VALUE" type="INT16S" writable="false" reportable="true" default="0x0000" optional="true" default="0">scaled value</attribute>
     <attribute side="server" code="0x0011" define="PRESSURE_MIN_SCALED_VALUE" type="INT16S" min="0x8001" max="0x7FFE" writable="false" optional="true" default="0">min scaled value</attribute>
     <attribute side="server" code="0x0012" define="PRESSURE_MAX_SCALED_VALUE" type="INT16S" min="0x8002" max="0x7FFF" writable="false" optional="true" default="0">max scaled value</attribute>
```

Comment:
Please also remove

>  min="0x8001" max="0x7FFE"

and 

> min="0x8002" max="0x7FFF"

This values are not part of spec. Unless you think it makes sense to keep them.